Database/init-dev-data.sql

```diff
@@ -9,6 +9,7 @@ INSERT INTO common.address ("id", "city", "country", "street", "state", "zip_cod
 (8, 'Kraków', 'Polska', 'ul. Floriańska 7', 'Małopolskie', '30-001'),
 (9, 'Gdańsk', 'Polska', 'ul. Długa 69', 'Pomorskie', '80-001'),
 (10, 'Wrocław', 'Polska', 'Legnicka 420', 'Dolnośląskie', '51-702');
+SELECT setval('common.address_id_seq', 10);
```

Comment:
I would suggest using SELECT to obtain the MAX index from table, instead of hardcoding it - in case you want to insert some more data, you won't have to remember to update the index in setval function.

Reply:
Updated